charts/refarch-templates/templates/hpa.yaml

```
@@ -14,7 +14,7 @@ spec:
   scaleTargetRef:
     apiVersion: apps/v1
     kind: Deployment
-    name: {{ $module.name }}
+    name: {{ $moduleName }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)